apple/Sources/FerrostarSwiftUI/Views/GridViews/NavigatingInnerGridView.swift

```diff
@@ -86,6 +96,29 @@ public struct NavigatingInnerGridView: View, CustomizableNavigatingInnerGridView
             },
             topCenter: { topCenter?() },
             topTrailing: {
+                // TODO: Extract to a separate view?
+                switch cameraControlState {
+                case .hidden:
+                    // Nothing
+                    EmptyView()
+                case let .showRecenter(onCenter):
+                    NavigationUIButton(action: onCenter) {
+                        Image(systemName: "location.north.fill")
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .frame(width: 18, height: 18)
+                    }
+                    .shadow(radius: 8)
+                case let .showRouteOverview(onRouteOverview):
+                    NavigationUIButton(action: onRouteOverview) {
+                        Image(systemName: "point.bottomleft.forward.to.point.topright.scurvepath")
+                            .resizable()
+                            .aspectRatio(contentMode: .fit)
+                            .frame(width: 18, height: 18)
+                    }
+                    .shadow(radius: 8)
+                }
```

Comment:
The important thing to note here is that we have moved the recenter button to be in the top corner. The logic behind that is that we *probably* only ever care about having one camera button visible at a time in my opinion (differing opinions welcome :D). So the camera button slot will either be a recenter button, a route overview button, or hidden, depending on state driven higher up.

Reply:
Discussed; we will move the recenter button to the bottom left corner to keep with convention.